cemu/tools/chocolateyinstall.ps1

```diff
@@ -4,22 +4,27 @@ $packageName = 'cemu'
 $url32       = 'http://cemu.info/releases/cemu_1.15.17.zip'
 $checksum32  = '317a34a15765e411d476464338d210eeca16b36c3e1401ea1f2096b0fa5a159f'
 $toolsPath   = Split-Path $MyInvocation.MyCommand.Definition
-$cemu_folder = "$toolsPath\cemu"
+$cemu_folder = "$(Get-ToolsLocation)\cemu"
 
 $packageArgs = @{
   packageName    = $packageName
-  url          = $url32
+  url            = $url32
   checksum       = $checksum32
   checksumType   = 'sha256'
   unzipLocation  = $toolsPath
 }
 
+If (Get-Item "$env:ChocolateyInstall\lib-bkp\cemu\tools\cemu\Cemu.exe" -ErrorAction SilentlyContinue) {
+	Write-Output "Migrating cemu location to $cemu_folder"
+	Move-Item "$env:ChocolateyInstall\lib-bkp\cemu\tools\cemu" "$(Get-ToolsLocation)" -Force
+	Remove-Item "$env:ChocolateyInstall\lib\cemu\tools\cemu" -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
Before running chocolateyinstall.ps1, chocolatey copies _known files_* from lib-bkp to lib. This is incomplete and worthless to us.

*: by known files I mean whatever files were extracted by chocolatey. Any files generated by cemu or the user are not included here.